orchagent/aclorch.cpp

```diff
@@ -122,6 +122,8 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
 
     sai_attribute_value_t value;
 
+    value.aclfield.enable = true;
```

Comment:
agree. having enable in this place is a bit a trick.
will move to rule creation